### PR TITLE
chore: documentation and comments for `--hotswap`

### PR DESCRIPTION
**Pull request by jayhelios**
----
Added a [couple comments](https://github.com/aws/aws-cdk/blob/7ccc644008d974c91bb789628d23f4f0b510075b/packages/aws-cdk/lib/api/hotswap/ecs-services.ts#L119) and a line to [this Readme](https://github.com/aws/aws-cdk/blob/7ccc644008d974c91bb789628d23f4f0b510075b/packages/aws-cdk/README.md) highlighting points from this [discussion](https://github.com/aws/aws-cdk/discussions/22241). Please feel free to adjust any wording, but if we are changing defaults for any parameters, even with hotswap, it should be documented somewhere. 

### All Submissions:

* [ ] Have you followed the guidelines in our [Contributing guide?](https://github.com/aws/aws-cdk/blob/main/CONTRIBUTING.md)

### Adding new Unconventional Dependencies:

* [ ] This PR adds new unconventional dependencies following the process described [here](https://github.com/aws/aws-cdk/blob/main/CONTRIBUTING.md/#adding-new-unconventional-dependencies)

### New Features

* [ ] Have you added the new feature to an [integration test](https://github.com/aws/aws-cdk/blob/main/INTEGRATION_TESTS.md)?
	* [ ] Did you use `yarn integ` to deploy the infrastructure and generate the snapshot (i.e. `yarn integ` without `--dry-run`)?

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

### CODE REVIEW

**Review by TheRealAmazonKendra — APPROVED** [review]

File: packages/aws-cdk/lib/api/hotswap/ecs-services.ts [hunk 1]
```diff
@@ -126,7 +126,8 @@ class EcsServiceHotswapOperation implements HotswapOperation {
         clusterPromises = [];
         servicePerClusterUpdates[clusterName] = clusterPromises;
       }
-
+// Forcing New Deployment and setting Minimum Healthy Percent to 0. 
+// As CDK HotSwap is development only, this seems the most efficient way to ensure all tasks are replaced immediately, regardless of original amount.
```
Comment (TheRealAmazonKendra) [inline packages/aws-cdk/lib/api/hotswap/ecs-services.ts:130]:
```suggestion
      // Forcing New Deployment and setting Minimum Healthy Percent to 0. 
      // As CDK HotSwap is development only, this seems the most efficient way to ensure all tasks are replaced immediately, regardless of original amount.
```

**Review by TheRealAmazonKendra — CHANGES_REQUESTED** [review]
Comments should be in line with code.

File: packages/aws-cdk/lib/api/hotswap/ecs-services.ts (hunk 1 above)
Comment (jayhelios) [reply packages/aws-cdk/lib/api/hotswap/ecs-services.ts:130]: Thanks for the correction!

**Comment by Naumel** [comment]
[action requested] Update the formatting:
```
aws-cdk: /codebuild/output/src290663950/src/github.com/aws/aws-cdk/packages/aws-cdk/lib/api/hotswap/ecs-services.ts
aws-cdk:   129:74  error  Trailing spaces not allowed  no-trailing-spaces
aws-cdk: âœ– 1 problem (1 error, 0 warnings)
aws-cdk:   1 error and 0 warnings potentially fixable with the `--fix` option.
```

@Mergifyio update

File: packages/aws-cdk/lib/api/hotswap/ecs-services.ts [hunk 2]
```diff
@@ -126,7 +126,8 @@ class EcsServiceHotswapOperation implements HotswapOperation {
         clusterPromises = [];
         servicePerClusterUpdates[clusterName] = clusterPromises;
       }
-
+      // Forcing New Deployment and setting Minimum Healthy Percent to 0.
```
Comment (TheRealAmazonKendra) [inline packages/aws-cdk/lib/api/hotswap/ecs-services.ts:129]:
```suggestion
      // Forcing New Deployment and setting Minimum Healthy Percent to 0.
```

**Review by TheRealAmazonKendra — APPROVED** [review]

**Pull request merged**